src/compas_timber/ghpython/components/CT_BTLx_Feature/metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "BTLx Feature",
+    "nickname": "BTLxFeature",
+    "category": "COMPAS Timber",
+    "subcategory": "Features",
+    "description": "Generates a feature from BTLx parameters.",
+    "exposure": 4,
+    "ghpython": {
+        "isAdvancedMode": true,
+        "iconDisplay": 0,
+        "inputParameters": [
+            {
+                "name": "Beam",
+                "description": "Beam to apply BTLx feature to.",
+                "typeHintID": "none",
+                "scriptParamAccess": 0
+            },
+            {
+                "name": "Reference Face",
```

Comment:
Maybe call this 'reference_side' or 'ref_side' instead so that it's coherent to the BTLx naming convention.